cmd/argocd-application-controller/commands/argocd_application_controller.go

```diff
@@ -206,6 +208,8 @@ func NewCommand() *cobra.Command {
 	command.Flags().BoolVar(&repoServerStrictTLS, "repo-server-strict-tls", env.ParseBoolFromEnv("ARGOCD_APPLICATION_CONTROLLER_REPO_SERVER_STRICT_TLS", false), "Whether to use strict validation of the TLS cert presented by the repo server")
 	command.Flags().StringSliceVar(&metricsAplicationLabels, "metrics-application-labels", []string{}, "List of Application labels that will be added to the argocd_application_labels metric")
 	command.Flags().StringVar(&otlpAddress, "otlp-address", env.StringFromEnv("ARGOCD_APPLICATION_CONTROLLER_OTLP_ADDRESS", ""), "OpenTelemetry collector address to send traces to")
+	command.Flags().BoolVar(&otlpInsecure, "otlp-insecure", env.ParseBoolFromEnv("ARGOCD_APPLICATION_CONTROLLER_OTLP_INSECURE", true), "OpenTelemetry collector insecure mode")
+	command.Flags().StringToStringVar(&otlpHeaders, "otlp-headers", env.ParseStringToStringFromEnv("ARGOCD_SERVER_OTLP_HEADERS", map[string]string{}, ","), "List of OpenTelemetry collector extra headers sent with traces, headers are comma-separated key-value pairs(e.g. key1=value1,key2=value2)")
```

Comment:
Can you add this to the deployment manifest [here](https://github.com/argoproj/argo-cd/blob/1600c03abafc4a0526d6909b3eddab5071d59508/manifests/base/application-controller/argocd-application-controller-statefulset.yaml#L140) and to the argocd cmd cm [here](https://github.com/argoproj/argo-cd/blob/1600c03abafc4a0526d6909b3eddab5071d59508/docs/operator-manual/argocd-cmd-params-cm.yaml#L20) and run codgen.

Reply:
@gdsoumya updated